galley/pkg/server/configmap_test.go

```diff
@@ -83,22 +119,36 @@ allowed:
 		t.Fatalf("Unexpected error: %v", err)
 	}
 
+	gotAddedFile := <-fake.added
+	if gotAddedFile != file {
+		t.Fatalf("watched wrong file: got %v want %v", gotAddedFile, file)
```

Comment:
consider rephrasing this- it may be confusing to read when debugging errors. That said, I'm struggling to find a better way to structure this.